examples/darwin-framework-tool/commands/interactive/InteractiveCommands.mm

```diff
@@ -30,9 +30,10 @@
 constexpr char kInteractiveModeInstruction[] = "╔══════════════════════════════════════════════════════════════════╗\n"
                                                "║                        Interactive Mode                          ║\n"
                                                "╠══════════════════════════════════════════════════════════════════╣\n"
-                                               "║ Stop and restart stack: [Ctrl+_] & [Ctrl+^ ]                     ║\n"
-                                               "║ Trigger exit(0)       : [Ctrl+@]                                 ║\n"
-                                               "║ Quit Interactive      : 'quit()' or `quit`                       ║\n"
+                                               "║ Stop and restart stack    : [Ctrl+_] & [Ctrl+^ ]                 ║\n"
+                                               "║ Suspend/Resume controllers: [Ctrl+Z]                             ║\n"
+                                               "║ Trigger exit(0)           : [Ctrl+@]                             ║\n"
+                                               "║ Quit Interactive          : 'quit()' or `quit`                   ║\n"
```

Comment:
While we're doing hot keys, it would be nice if Ctrl-D did a quit like how it does in shell.

Reply:
I can try that in a follow up as it seems a new shortcut and I like to keep PR as small as possible. And additionally I took a quick try at it and I have not yet understood why it fails on first try 😗